### DISCUSSION TITLE
Multimount setup from Trophy Lodge

### DISCUSSION THREAD

**Discussion by Foefaux** [janstehno/cotw-companion#97] [Ideas]
Currently there is no way to add multi-trophy mounts to the trophy lodge book. Any animal that could be used in a mount is counted towards the completion of the mount but there isn’t a way to remove animals from the eligible list of animals to make into multi mounts once they’ve already been made into one.

**Comment by janstehno** [comment]
For now the multimounts feature just shows you what multimounts you can make from animals in the trophy lodge. Nothing else. I have plans to further expand it in the future.
